### PR TITLE
LPS-95300 Create a configuration options to limit the size and type of kb attachments

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

**Comment by robertoDiaz** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
Just started reviewing :)

File: modules/apps/knowledge-base/knowledge-base-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1,3 +1,6 @@
+allowed-knowledge-base-attachment-mime-types=Allowed Knowledge Base attachment mime types
+allowed-knowledge-base-attachment-mime-types-description=Set valid mime types for Knowledge Base attachments. A mime type of * will permit all mime types.
+allowed-wiki-attachment-mime-types-description=Set valid mime types for wiki attachments.
```
Comment (robertoDiaz) [inline modules/apps/knowledge-base/knowledge-base-api/src/main/resources/content/Language.properties]: typo!! my bad 👎

**Comment by robertoDiaz** [comment]
cierro y actualizo

**Comment by adolfopa** [comment]
Just started reviewing :)

**Comment by adolfopa** [comment]
Pull request submitted to https://github.com/robertoDiaz/liferay-portal/pull/815. [See changes here.](https://github.com/adolfopa/liferay-portal/compare/960e2b723bcc~6...960e2b723bcc)

**Pull request closed without merging**